homeassistant/components/smhi/weather.py

```diff
@@ -139,7 +148,25 @@ def __init__(
             configuration_url="http://opendata.smhi.se/apidocs/metfcst/parameters.html",
         )
         self._attr_condition = None
-        self._attr_temperature = None
+        self._attr_native_temperature = None
+
+    @property
+    def extra_state_attributes(self) -> Mapping[str, Any] | None:
+        """Return additional attributes."""
+        if self._forecasts:
+            wind_gust = self._forecasts[0].wind_gust
+            if (wind_speed_unit := self._weather_option_wind_speed_unit) is not None:
```

Comment:
I think this should be `self._wind_speed_unit`

Reply:
As discussed will fix this